composer.json

```diff
@@ -6,39 +6,40 @@
     "require": {
         "php": "^7.1.3",
         "ext-pdo_sqlite": "*",
-        "doctrine/doctrine-fixtures-bundle": "^2.3",
-        "erusev/parsedown": "^1.6",
-        "ezyang/htmlpurifier": "^4.9",
-        "sensio/framework-extra-bundle": "^3.0",
-        "sensiolabs/security-checker": "^4.1",
-        "symfony/asset": "^3.3",
-        "symfony/expression-language": "^3.3",
-        "symfony/flex": "^1.0",
-        "symfony/form": "^3.3",
-        "symfony/framework-bundle": "^3.3",
-        "symfony/monolog-bundle": "^3.1",
-        "symfony/orm-pack": "^1.0",
-        "symfony/polyfill-apcu": "^1.4",
-        "symfony/security-bundle": "^3.3",
-        "symfony/swiftmailer-bundle": "^3.0",
-        "symfony/translation": "^3.3",
-        "symfony/validator": "^3.3",
-        "symfony/yaml": "^3.3",
-        "twig/extensions": "^1.5",
+        "doctrine/doctrine-fixtures-bundle": "2.4.0",
+        "erusev/parsedown": "1.6.3",
```

Comment:
Why exact requirements everywhere ? This is clearly no a best practice (forbidding to update patch releases with a simple `composer update` makes maintenance harder)

Reply:
NVM, It's a temporal change to avoid update these dependencies right now, later these constraints will be reverted.

Reply:
But 3/4 of the dependencies are not in the composer.json (as they are transitive deps), and these are still updated to dev versions. So I don't think these changes make sense (and they make the review harder, as well as the future maintenance)

Reply:
> these are still updated to dev versions.

That's why I added `"symfony/lts": "dev-master"` to avoid transitive deps update to 4-dev, all of them are forced to 3-dev state.

> So I don't think these changes make sense (and they make the review harder, as well as the future maintenance)

I guess you're refering to changes like this `"erusev/parsedown": "1.6.3"`? if so, there are temporal changes to avoid update these deps right now. This `"minimum-stability": "dev"` option forces all deps to dev unless there is a constrains that prevents it (that's what I did with Symfony deps ``"symfony/lts": "dev-master"`). 

I did these changes to rely on Symfony 3/4 dev deps only. I think isn't a good idea rely on all dev deps during the migration to 4.0, since we would be testing against dev deps that later they might not work.

When the upgrade finishes, we can go back safely to normal constraints `1.6.3` > `^1.6`, remove `"minimum-stability": "dev"` option and fix the Symfony release deps.

If it isn't reasonable, please, what would be the alternative during this upgrade?